catalog-info.yaml

```diff
@@ -287,3 +287,93 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-beats-libbeat
+  description: "Beats libbeat pipeline"
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-libbeat
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: beats-libbeat
+      description: "Beats libbeat pipeline"
+    spec:
+#      branch_configuration: "main 7.* 8.* v7.* v8.*" TODO: temporarily commented to build PRs from forks
```

Comment:
I'm not sure what's the reason for using `v7.* v8.*` but as far as I know branches are not using `vx.y.z` but tags. Is branch_configuration also used for tags?

`7.*` is not required but `7.17`

```suggestion
#      branch_configuration: "main 7.17 8.*" TODO: temporarily commented to build PRs from forks
```

Reply:
I've asked the question in the BK channel, and it looks like we have some right options here and we have to choose one of them and then use this option across all pipelines of the `beats` repo. I think we have enough time to think about it and I do believe we will have a strong opinion regarding this option and other ones as well at the end of the main part of the migration of the pipelines, right before the migration of the main pipeline... I suggest keeping it as it is right now and back to the topic again a bit later. @v1v, WDYT?